torch/csrc/jit/runtime/profiling_graph_executor_impl.h

```diff
@@ -14,6 +14,16 @@ struct ProfilingGraphExecutorImpl : public GraphExecutorImplBase {
   GraphExecutorState getDebugState() override;
   ~ProfilingGraphExecutorImpl() override = default;
 
+  void flushCompilationCache() {
+    pr_.reset();
+    fallback_plan_.reset();
```

Comment:
I think some of these have mutexes that you're not accessing

Reply:
Ah, yes! Thank you for pointing this out!